data/postgres.py

```diff
@@ -1,15 +1,19 @@
+from data.abstract import DataManager
 import psycopg2
 
-class PostgresDataBase:
+class PostgresDataBase(DataManager):
     def __init__(self):
         print("Transfering data to postgres data base")
+        self.conn = psycopg2.connect(host = "localhost", dbname = "postgres", user = "postgres", password = "toster1111", port = 5432)
+        self.conn.set_session(autocommit=True)
 
-    def transfer_data(self, price, date, ema_long, ema_short, macd, rsi, adx):
-        conn = psycopg2.connect(host = "localhost", dbname = "postgres", user = "postgres", password = "toster1111", port = 5432)
-        cur = conn.cursor()
+    def insert(self, price, date, ema_long, ema_short, macd, rsi, adx):
         insert_values = 'INSERT INTO crazy_tr8 (price, date, ema_long, ema_short, macd, rsi, adx) VALUES (%s, %s, %s, %s, %s, %s, %s)'
```

Comment:
Crazy_tr8 should not be a table but a schema.

Usually we should let the user create a schema and choose which user he/she wants to use. We should also require that the schema already has all the tables (like we do here). I think we should implement well established approach with dealing with database. Some responsibilities are on user side some on ours:

1. We should provide create_schema.sql which user can use to create database
2. User should setup database and user ans the schema on his/her own and provide connection string when using our app for example: (postgres://postgres:toster1111@localhost:5432/crazy_tr8)

To sum up the flow is:

1. We are committing create_schema.sql file to repo
2. User is creating database using for example:

psql -u postgres -c "create database crazy_tr8"

3. User run our create_schema.sql file on newly created dB:

psql -u postgres -d crazy_tr8 -f create_schema.sql

4. User starts app with -datasource postgres://postgres:toster1111@localhost:5432/crazy_tr8